graphql-dgs-spring-webflux-autoconfigure/src/main/kotlin/com/netflix/graphql/dgs/webflux/autoconfiguration/DgsWebFluxAutoConfiguration.kt

```diff
@@ -190,11 +190,13 @@ open class DgsWebFluxAutoConfiguration(private val configProps: DgsWebfluxConfig
     }
 
     @Bean
+    @ConditionalOnMissingBean
```

Comment:
I added @ConditionalOnMissingBean in both places out of caution, in case someone does happen to include both DgsWebFluxAutoConfiguration and DgsAutoConfiguration; in fact, a few of the integration tests were importing both.